### PR TITLE
Add Publiclab organization link to community-toolbox website 

### PR DESCRIPTION
**Pull request by DakshGondaliya**
fixed:  https://github.com/publiclab/plots2/issues/3589

### CODE REVIEW

**Comment by DakshGondaliya** [comment]
@jywarren, please review

File: index.html [hunk 1]
```diff
@@ -136,10 +142,18 @@ <h2 class="contributors-head">Contributors</h2>
...
-        - Open Source
+      <p class="links">
+          Page powered by <a href="https://github.com/publiclab/community-toolbox"><i class="fa fa-github"></i></a>
+          <a href="https://github.com/publiclab/community-toolbox">Commmunity Toolbox</a> 
+          - Open Source
+          &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;
+  
+          <a href="https://github.com/publiclab"><i class="fa fa-github"></i></a>
+          <a href="https://github.com/publiclab">Public Lab</a> 
+          &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;
```
Comment (jywarren) [inline index.html]: Hmm, is it possible to do this via inline CSS instead of all these? if not, no big deal!

**Comment by DakshGondaliya** [comment]
com
com full

**Comment by DakshGondaliya** [comment]
Please review it again @jywarren @SidharthBansal @igniteeng000

**Comment by jywarren** [comment]
This looks great. Maybe just remove the word "GitHub" but leave the icon?
It kind of looks like GitHub is an author, but we want it to indicate the
GitHub page of Publiclab, right?

Such awesome work! Thank you so much!

**Comment by jywarren** [comment]
Hi -- this looks perfect; merging now. 👍 🎉  If you're looking for a new challenge, please take a look at https://code.publiclab.org#r=all; we prefer to leave `first-timers-only` issues for complete newcomers, but there are plenty of others we need help with! Thanks!

**Pull request merged**